### DISCUSSION TITLE
Server: reduce lag and delays for transmitions

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1593] [Q&A]
### Description

i hope you understand my point guys by testing it: so, as we all know, teamtalk is really has grate transmition for voice and others, it's bettar than all other calling apps, teamtalk is the best, but the only problem is, that it has a really little delay when even your internet is really fast and the server is powerful: for example, let me tell you how i made sure of this: i made a teamtalk server with localhost in my laptop, and i logged in to it my self, and enabled hear my self option to hear my self: when i was talking, i had a really tiny delay, and my ping was about 5, or 7 MS or something around this: i even mannaged to test this delay with 2 MS ping, but it stil did have a little delay: i expect that if my ping is 2 ms or something like that, i should hear my self as soon as i start talking: can you try it: server vertion: 5.11: tested with classic and Qt clients and both had this problem with the delay.

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [X] TeamTalkServer

**Comment by ChrisDuffley** [comment]
Try changing the Transmit Interval in the Update Channel dialog to 20 and the Frame Size to 20. This should reduce it by at least half. Depends on people's connections of course.

Hope this helps!

**Comment by amirmahdifard** [comment]
@ChrisDuffley thanks, i saw this option when i was updating channels befor, so can you please tell me what is this option used for? even completly?

**Comment by amirmahdifard** [comment]
@ChrisDuffley oh! i just tryed these 2 options because i want to figger them out that what is them, and i changed the frame size to 2.5 mseck and the voices became really strange lol, but i don't know what you can really do with them, please explane to me if you know, and by the way, when i did this, that delay thing is stil remaning that needs fix, but may be it reduced a very little bit.
